Sources/Swiftly/ListAvailable.swift

```diff
@@ -95,18 +99,11 @@ struct ListAvailable: SwiftlyCommand {
                 printToolchain(toolchain)
             }
         } else {
-            print("Available release toolchains")
+            print("Available toolchains")
             print("----------------------------")
-            for toolchain in toolchains where toolchain.isStableRelease() {
+            for toolchain in toolchains {
```

Comment:
Good call don't need to separate these two lists

Reply:
This else only gets invoked in the absence of a selector, which defaults to all of the releases, since every snapshot build for every branch would be overwhelming and probably not very useful.

I can retain the "release" toolchains label to make it clear that these are only releases, but the where clause shouldn't be necessary here.